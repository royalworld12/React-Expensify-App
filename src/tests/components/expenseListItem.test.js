import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../components/expenseListItem";
import expenses from '../fixtures/expenses';

test('Should render expenseListItem with expenses',()=>{
    const wrapper = shallow(<ExpenseListItem {...expenses[1]}/>)
    expect(wrapper).toMatchSnapshot();
})