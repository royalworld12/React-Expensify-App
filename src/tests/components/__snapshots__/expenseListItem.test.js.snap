// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render expenseListItem with expenses 1`] = `
<div>
  <Link
    replace={false}
    to="edit/2"
  >
    rent
  </Link>
  <p>
    109500
     - 
    -345600000
  </p>
</div>
`;
